ui/src/app/job-list/job-list.component.html

```diff
@@ -23,6 +23,10 @@ <h2 class="settings-header">Displayed columns</h2>
                       (click)="$event.stopPropagation()">{{ df.display }}</mat-checkbox>
       </mat-list-item>
     </mat-list>
+    <div *ngIf="settingsService.getSavedSettingValue('hideArchived', projectId) != null">
```

Comment:
nit: I would generally consider this too heavy to put in a template, would probably extract this into a helper method on the component.